hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/service/MonitorService.java

```diff
@@ -179,8 +180,8 @@ private static MonitorServiceConfiguration buildRuntimeConfiguration(
                         port = socketBinding.getAbsolutePort();
                     } else {
                         OutboundSocketBinding serverBinding = serverOutboundSocketBindingValue.getValue();
-                        address = WildflyCompatibilityUtils.
-                                outboundSocketBindingGetResolvedDestinationAddress(serverBinding).getHostName();
+                        address = WildflyCompatibilityUtils
+                                .outboundSocketBindingGetResolvedDestinationAddress(serverBinding).getHostName();
```

Comment:
these lines I didn't touch. But the code formatter is reformatting them. I don't know why our formatters are different. But this trivial is just formatting, no structural change.